specs/DesignDiscussions/InputLabels.md

```diff
@@ -91,6 +129,42 @@ If we want to allow the host to control formatting of the labels, either in gene
 ```
 Another option is to not put this in host config, and instead control this styling via support of native styling on required and optional labels (i.e. CSS, Xaml Styles, etc). This decision speaks to a larger discussion of the direction of host config that is beyond the scope of this document.
 
+> ### Host side styling
+> 
+> Following the previously mentioned text formatting the proposed set of properties we could allow the host application to override would be almost the same set of properties we provide for configuring a TextBlock: 
+> 
+> | Property | Default Value |HostConfig | Native |
+> | --- | --- | --- | --- |
+> | spacing | "default" | X | |
+> | fontType | "default" | X | X |
+> | size | "default" | X | X |
+> | weight | "default" | X | X |
+> | color | "default" | X | X |
+> | isSubtle | false | X | |
+> | separator | false | X | |
+>
+> ```json
+> "inputLabelFormatting": 
+> {
+>	"requiredInputs": {
+>		"spacing": "large",
+>		"weight": "Bolder",
+>		"fontType": "monospace",
+>		"size": "medium",
+>		"color": "good"
+>	},
+>	"optionalInputs": {
+>		"isSubtle": true
+>	}	
+> }
+> ```
+>
+> While it's mentioned that we should allow almost all TextBlock properties to be modified by the label property, some of them must stay out of bounds for hosts as they would provide a bad user experience:
```

Comment:
>hem must stay out of bounds for hosts as they would provide a bad user experien [](start = 123, length = 79)

Great -- should this metapoint around ensuring we arent enabling poor end-user card experience, be included in the HostConfig discussion also?